### PR TITLE
Update register_types in binding_to_external_libraries.rst

### PR DESCRIPTION
**Pull request by hugarty**
Based in https://github.com/godotengine/godot/issues/84456

Change register_tts_types() for initialize_tts_module() 
Change unregister_tts_types() for uninitialize_tts_module()

### CODE REVIEW

File: contributing/development/core_and_modules/binding_to_external_libraries.rst [hunk 1]
```diff
@@ -85,8 +85,8 @@ These files should contain the following:
 
     /* register_types.h */
 
-    void register_tts_types();
-    void unregister_tts_types();
+    void initialize_tts_module();
```
Comment (AThousandShips) [inline contributing/development/core_and_modules/binding_to_external_libraries.rst:88]:
```suggestion
    void initialize_tts_module(ModuleInitializationLevel p_level);
```
Same below

File: contributing/development/core_and_modules/binding_to_external_libraries.rst [hunk 2]
```diff
@@ -98,12 +98,15 @@ These files should contain the following:
...
+        if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE) {
+            return;
+        }
         ClassDB::register_class<TTS>();
     }
 
-    void unregister_tts_types() {
-        // Nothing to do here in this example.
+    void uninitialize_tts_module(ModuleInitializationLevel p_level) {
+	    // Nothing to do here in this example.
```
Comment (AThousandShips) [inline contributing/development/core_and_modules/binding_to_external_libraries.rst:109]:
```suggestion
        // Nothing to do here in this example.
```
Spaces changed to tabs

**Comment by hugarty** [comment]
Related to [these comment](https://github.com/godotengine/godot/issues/84456#issuecomment-1798139905)

What I should do ?

**Comment by AThousandShips** [comment]
Please restore the spaces, it should be spaces and not tabs

**Review by AThousandShips — APPROVED** [review]
LGTM

Can you please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1.

**Pull request merged**